### PR TITLE
Refactor large 2d flow

### PR DESCRIPTION
**Pull request by annshress**
Addresses - https://github.com/niaid/image_portal_workflows/issues/292

Depends on - https://github.com/niaid/image_portal_workflows/tree/czi_to_zarr_wf

### This PR doesn't introduce any:

- [x] Binary files
- [x] Temporary files, auto-generated files
- [x] Secret keys
- [x] Local debugging `print` statements
- [x] Unwanted comments (e.g: \# Gets user from environment for code `os.environ['user']` )

### This PR contains valid:

- [ ] tests

### CODE REVIEW

File: em_workflows/lrg_2d_rgb/flow.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def bioformats_gen_zarr(file_path: FilePath):
         LRG2DConfig.bioformats2raw,
         "--overwrite",
         "--scale-format-string",
-        "%2$d",
+        "'%2$d'",
```
Comment (blowekamp) [inline em_workflows/lrg_2d_rgb/flow.py:60]: I do not believe that the command goes through a shell, so the quotes are not needed here.

File: em_workflows/lrg_2d_rgb/flow.py [hunk 2]
```diff
@@ -71,39 +71,42 @@ def bioformats_gen_zarr(file_path: FilePath):
         output_zarr,
     ]
     FilePath.run(cmd, log_fp)
-    cmd = ["zarr_rechunk", "--chunk-size", "512", output_zarr]
-    log_fp = f"{file_path.working_dir}/{file_path.base}_rechunk.log"
-    FilePath.run(cmd, log_fp)
+    zarr_images = HedwigZarrImages(zarr_path=Path(output_zarr), read_only=False)
+    zarr_image: HedwigZarrImage = next(zarr_images.series())[1]
```
Comment (philipmac) [inline em_workflows/lrg_2d_rgb/flow.py:75]: this line is a bit cryptic - can we write a reminder that the `.series()` method is returning a tuple, the second elt of which is the object, thus requiring us to walk one element in (and ignore the first one)? I remember this now, but I'm likely to forget this later...

Comment (blowekamp) [reply em_workflows/lrg_2d_rgb/flow.py:75]: This could alternatively be something like: `zarr_image = zarr_images[zarr_images.get_series_keys()[0]]`

Comment (annshress) [reply em_workflows/lrg_2d_rgb/flow.py]: Yes Mike. Although it was clear enough, I thought maybe typing can help code navigation.

Comment (philipmac) [reply em_workflows/lrg_2d_rgb/flow.py]: yeah, I like the type hinting. It's cute.

Comment (annshress) [reply em_workflows/lrg_2d_rgb/flow.py]: I agree! It's better if used in a controlled manner.

**Review by philipmac — APPROVED** [review]

**Pull request merged**